Packs/FeedRecordedFuture/Integrations/FeedRecordedFuture/FeedRecordedFuture.py

```diff
@@ -403,7 +403,14 @@ def main():
             indicators = fetch_indicators_command(client, client.indicator_type)
             # we submit the indicators in batches
             for b in batch(indicators, batch_size=2000):
-                demisto.createIndicators(b)
+                non_duplicates_dict: Dict[str, Dict] = dict()
+                for indicator in b:
+                    if indicator_value := indicator.get("value"):
+                        # each value is added to the dict only ones
+                        if not non_duplicates_dict.get(str(indicator_value).lower()):
+                            non_duplicates_dict[str(indicator_value.lower())] = indicator
+                unique_indicators_list = list(non_duplicates_dict.values())
```

Comment:
this should be a temporary fix or a permanent one? either way, add in a comment link to the relevant issue since we don't de-dup in every integration.